packages/ui/src/2_molecules/RadioButton/RadioButton.tsx

```diff
@@ -7,25 +7,38 @@ import { IRadioButtonProps } from './RadioButton.types';
 
 export const RadioButton = forwardRef<HTMLInputElement, IRadioButtonProps>(
   (
-    { id, label, disabled, name, className, dataLayoutId, labelInfo, ...rest },
+    {
+      id,
+      label,
+      disabled,
+      name,
+      className,
+      dataLayoutId,
+      labelInfo,
+      contentToShow,
+      ...rest
+    },
     ref,
   ) => (
-    <div
-      className={classNames(styles.radioButton, disabled && styles.disabled)}
-    >
-      <label htmlFor={id} className={classNames(styles.label, className)}>
-        <input
-          id={id}
-          type="radio"
-          disabled={disabled}
-          data-layout-id={dataLayoutId}
-          ref={ref as LegacyRef<HTMLInputElement>}
-          name={name}
-          {...rest}
-        />
-        {label}
-      </label>
-      <div>{labelInfo}</div>
-    </div>
+    <>
+      <div
+        className={classNames(styles.radioButton, disabled && styles.disabled)}
+      >
+        <label htmlFor={id} className={classNames(styles.label, className)}>
+          <input
+            id={id}
+            type="radio"
+            disabled={disabled}
+            data-layout-id={dataLayoutId}
```

Comment:
```suggestion
            {...applyDataAttr(dataLayoutId)}
```
Switch this to use new utility function for applying layout id attribute, examples can be found in `SimpleTableRow` or other components